src/main/java/nl/tudelft/lifetiles/graph/models/DefaultGraphParser.java

```diff
@@ -106,9 +123,14 @@ private void parseEdges(final String filename,
     @Override
     public final Graph<SequenceSegment> parseFile(final String filename,
             final GraphFactory<SequenceSegment> gfact) {
+        Calendar cal = Calendar.getInstance();
+        long startTime = cal.getTimeInMillis();
         Graph<SequenceSegment> graph = gfact.getGraph();
         parseVertices(filename, graph);
         parseEdges(filename, graph);
+        Calendar cal2 = Calendar.getInstance();
+        System.out.println("Graph parsed. Took "
```

Comment:
The runtime measurement should not live in this method since it has nothing to do with parsing a file.